omero_figure/urls.py

```diff
@@ -33,6 +33,11 @@
     path('imgData/<int:image_id>/', views.img_data_json,
          name='figure_imgData'),
 
+    re_path(r'^max_projection_range_exceeded/'
+            r'(?P<iid>[0-9]+)/(?:(?P<z>[0-9]+)/)?(?:(?P<t>[0-9]+)/)?$',
+            views.max_projection_range_exceeded,
+            name='max_projection_range_exceeded'),
```

Comment:
I don't know that the url `path` format allows for optional parameters as we have in the regex here. https://docs.djangoproject.com/en/5.1/ref/urls/#django.urls.path. Tried searching and this suggests that it's not supported. Since we know that the re_path works, let's stick with that for now.